scripts/stackstorm.js

```diff
@@ -254,8 +254,25 @@ module.exports = function(robot) {
       if (execution_details) {
         args.push(util.format('Execution details available at: %s', execution_details));
       }
-
-      robot.messageRoom.apply(robot, args);
+      
+      if (robot.adapterName == 'slack') {
+        var attachment_color = 'dfdfdf';
+        if (data.message.indexOf("status : failed") > -1) {
+          attachment_color = 'danger';
+        }
```

Comment:
Like I mentioned on slack, it would be cool to use color green for actions that succeed. 76EE00 is the hex code. Can you make a fix?